spmk-musica-ecclesiastica.csl

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" default-locale="pl-PL">
+  <info>
+    <title>Styl SPMK Musica Ecclesiastica (SPMK Musica Ecclesiastica)</title>
+    <id>http://www.zotero.org/styles/spmk-musica-ecclesiastica</id>
+    <link href="http://www.zotero.org/styles/spmk-musica-ecclesiastica" rel="self"/>
+    <author>
```

Comment:
Please add:
1. a link to the style you used as a template here
2. the link to the guidelines with `rel="documentation`.

Both are requirements and explained here: https://github.com/citation-style-language/styles/blob/master/STYLE_REQUIREMENTS.md